assets/css/index.css

```diff
@@ -211,75 +217,316 @@
   background-color: var(--editor-tab-bg-hover);
 } */
+
 .tab-close, .wksp-tab-close {
   height: auto;
   margin: 0 0.5em 0 1em;
   width: 0.45vw!important;
   color: var(--display-4-color);
 }
+
 .tab-close:hover, .wksp-tab-close:hover {
   filter: opacity(0.5);
 }
+
 #wksp-tab-add {
   justify-content: center;
   align-items: center;
   width: 100%;
   margin: 0;
 }
+
 #tab-add {
   cursor: pointer;
   margin-right: 0;
 }
+
 #tab-add:hover {
   background-color: inherit;
 }
+
 .btn {
   cursor: pointer;
 }
 
 body {
   overflow-x: hidden;
-  background-color: var(--document-body-bg);
+  background-color: var(--navbar-upper-bg);
+  height: 100vh;
+}
+
+.findall {
+  width: 400px;
+  height: 500px;
+  position: absolute;
+  top: 25vh;
+  left: 25vw;
+  z-index: 100;
+  display: none;
+  opacity: 0;
+  transition: opacity 0.3s ease 0s;
+  background: rgba(255, 255, 255, 0.7) none repeat scroll 0% 0%;
+  border: 2px solid var(--display-4-color);
+  border-radius: 5px;
+}
+
+.findalltop {
+  width: 100%;
+  height: 2.5%;
+  background: #fff
+}
+
+.findallbkgd {
+  width: 100%;
+  height: 95%;
+  filter: blur(10px);
+  z-index: 99;
+  background: rgba(255, 255, 255, 0.65);
+}
+
+.findallbody {
+  width: 100%;
+  height: 95%;
+  filter: blur(0px);
+  position: absolute;
+  top: 2.5%;
+  left: 0;
+  z-index: 101;
+  background: transparent;
+  display: flex;
+  align-items: center;
+  padding-top: 20px;
+  justify-content: flex-start;
+  flex-direction: column;
+  overflow-x: hidden;
+  overflow-y: scroll
+}
+
+.findallwindow {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  width: 95%;
+  height: 10%;
+}
+
+.fa.fa-search {
+  margin-right: 15px;
+  font-size: 1rem;
+}
+
+#findallsearch {
+  width: 85%;
+  font-family: Open Sans;
+  background: transparent;
+  border: 1px solid #888;
+  border-radius: 5px;
+  padding-left: 10px;
+  height: 100%;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+#findallstatus {
+  font-size: 14px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  width: 90%;
+  margin-top: 15px;
+  margin-bottom: 5px;
+}
+
+#overlay_top {
+  background-color: var(--navbar-upper-bg);
+}
+
+#mainview {
+  width: 100%;
+  height: 100%;
+  margin-right: auto;
+  margin-left: auto;
+  transition: filter 0.75s
+}
+
+#header-flex {
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+#header-row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.display-4.text-center.text-muted.title {
+  font-size: calc(0.5rem + 1.3vw + 0.5vh);
+  width: auto;
+  margin: 0 10vw 0 10vw
+}
+
+.download-div {
+  width: 10%;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  align-items: center"
+
```

Comment:
Typo here?  I think it's supposed to be a semicolon after "center", not a double-quote.  There also seems to be an extra newline.